modules/gdscript/tests/scripts/analyzer/features/parent_class_methods_indentified.gd

```diff
@@ -0,0 +1,18 @@
+extends Node
+
+class A extends Node:
+    func f1():
+        pass
+
+class B extends A:
+    signal test_signal()
+
+    func f2():
+        pass
+
+    func test_connect():
+        test_signal.connect(f1)
+        test_signal.connect(f2)
+
+func test():
+    B.new().test_connect()
```

Comment:
You're leaking `B` here, it needs to be freed (either manually, or by adding it to the scene tree).